examples/scale/frontend/frontend.go

```diff
@@ -67,31 +70,58 @@ func Run() {
 }
 
 func doCreate(cfg config.View, fe pb.FrontendClient) {
```

Comment:
I'm not a fan of "do" being in function names.  It's like appending "Value" to the end of variable names.